ui/app/i18n/registration/locale_es.json

```diff
@@ -88,6 +88,8 @@
     "MESSAGE_DIALOG_OPTION_OKAY": "OK",
     "NO_LOCATION_TAGGED_TO_VISIT_LOCATION": "No hay ubicación puesta para la Ubicacion de la Visita Encontrada",
     "REGISTRATION_ADDTIONAL_IDENTIFIERS": "Identificadores Adicionales",
+    "VISIT_OF_THIS_PATIENT_AT_SAME_LOCATION_EXISTS": "Ya existe la visita de este paciente en la misma ubicación",
```

Comment:
ONLY English translations need to be checked in. Not in other languages. Those will come automatically from Transifex. See this: https://bahmni.atlassian.net/wiki/spaces/BAH/pages/41287695/Dev+Notes+for+Translating+Bahmni